docs/source/en/quicktour.md

```diff
@@ -173,11 +173,11 @@ The model configuration is a 🧊 frozen 🧊 dictionary, which means those para
 
 Some of the most important parameters are:
 
-* `sample_size`: the height and width dimension of the input sample.
-* `in_channels`: the number of input channels of the input sample.
-* `down_block_types` and `up_block_types`: the type of down- and upsampling blocks used to create the UNet architecture.
-* `block_out_channels`: the number of output channels of the downsampling blocks; also used in reverse order for the number of input channels of the upsampling blocks.
-* `layers_per_block`: the number of ResNet blocks present in each UNet block.
+* `sample_size`: The height and width dimension of the input sample.
+* `in_channels`: The number of input channels of the input sample.
+* `down_block_types` and `up_block_types`: The type of down- and upsampling blocks used to create the UNet architecture.
+* `block_out_channels`: The number of output channels of the downsampling blocks; also used in reverse order for the number of input channels of the upsampling blocks.
+* `layers_per_block`: The number of ResNet blocks present in each UNet block.
```

Comment:
I don't think we need to capitalize these; it's another stylistic choice :)

Reply:
Reversed.